tech.json

```diff
@@ -853,6 +853,17 @@
             "Can be used to make it through an opening door, or barely just past a ledge."
           ]
         },
+        {
+          "name": "canAbuseReserveRefill",
+          "requires": [ "ReserveTank" ],
+          "note": "Abusing the reserve energy refill ability to pause game time but continue running iframes."
```

Comment:
This needs to require some way to show that Samus loses all of her normal energy to perform said strat. (I think it's probably safe to assume that by the time you're performing these types of tricks, there is a way to damage down to the correct energy range.) 

I was looking at the Big Boy Room, and there is:
- "requires": [{"energyAtMost": 1}]

Would it work if this tech required {"energyAtMost": 0}, to indicate that Samus has run out of energy, then have the strats that use it look something like this, indicating 1 hit before the reserve, then 1 after?

{"enemyDamage": {"enemy": "Sova","hits": 1,"type": "contact"}},
"canAbuseReserveRefill",
{"enemyDamage": {"enemy": "Sova","hits": 1,"type": "contact"}}

Reply:
Not indicating the need to damage down is a problem.  
'Reserves being full' also isn't handled adequately.
I'll think on this some more and move this part to another PR.

Reply:
After merging this, I'm thinking that the reserve strats are actually really close to being correct and easily fixable. Using what I shared above does imply two hits. One to get Samus to 0 energy, included in the canAbuseReserveRefill, and then the second after it.

"canAbuseReserveRefill",
{"enemyDamage": {"enemy": "Sova","hits": 1,"type": "contact"}}

I don't think we need a way to represent getting down to <50 energy before that, except in notes. The logical requirements would still be right. 

The reserve capacity requirements can actually be deleted. The strat will only be in logic if the player has enough reserve energy. Just the same as strats can have damage requirements with normal energy - it's only in logic if there's enough energy.